src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -86,6 +90,14 @@ trait WorkspaceService extends HttpService with PerRequestCreator with FireCloud
             }
           }
         } ~
+        path("updateAttributes") {
+          patch {
+            entity(as[Seq[AttributeUpdateOperation]]) { attributeOp =>
```

Comment:
I'm almost done addressing the comments, but just a tiny bit confused on this part since I couldn't find any examples of it in the code. If we're just going to pass it through without caring about what it is so rawls can deal with it, what would be the entity type here? Would it be String? i.e.:

```
        path("updateAttributes") {
          patch {
            entity(as[String]) { attributeOp =>
              requestContext =>
                externalHttpPerRequest(requestContext, Patch(workspacePath, attributeOp))
            }
          }
        }
```
Or would it be some sort of wildcard type?